files/autostop/stop.sh

```diff
@@ -6,4 +6,8 @@ if isTrue "${DEBUG_AUTOSTOP}"; then
 fi
 
 logAutostopAction "Stopping Java process"
-kill -SIGTERM 1
+if isTrue "${AUTOSTOP_USE_PKILL}"; then
+  sudo pkill -f --signal SIGTERM mc-server-runner
```

Comment:
I'm wondering if pkill should just be used always since it'll work in the typical case. 

However, is sudo actually needed?

Reply:
It is in the environment I am currently using. I could put it behind a flag though if you want?

Reply:
Yeah, putting that behind a flag and then using the pkill approach unconditionally would be great.